test/tools/libtesteth/TestHelper.cpp

```diff
@@ -501,6 +501,7 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 			json_spirit::read_string(s, v);
 			doTests(v, true);
 			addClientInfo(v, testfilename);
+			postParseJson(v);
```

Comment:
To be honest, I don't think `doTests` should modify the input document, but return an output JSON file when it's doing filling.  Moreover, `doTests` should be responsible for the correct formatting of the output.   That's for a separate PR.